### PR TITLE
Tune Docker image for server side and include newer GDAL (Ubuntu 20.04)

### PR DESCRIPTION
**Pull request by vikoivun**

### CODE REVIEW

File: deploy/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,2 @@
 -r ../requirements.txt
-uwsgi
```
Comment (mhieta) [inline deploy/requirements.txt:2]: This needs to be added also to `requirements.in`-file, because `requirements.txt` is derived from that file.

**Review by mhieta — CHANGES_REQUESTED** [review]
@vikoivun We should merge on top of `develop`-branch, since it has the latest changes we want with GeoDjango 3 and GDAL 3 for example.

File: deploy/requirements.txt (hunk 1 above)
Comment (vikoivun) [reply deploy/requirements.txt:2]: This deploy/requirements.txt. It is not handled by pip-tools. Although it should, as whitenoise can change in incompatible ways.
Could we maybe just make whitenoise part the smbackend proper? It should work just fine in local development as well.

Comment (mhieta) [reply deploy/requirements.txt:2]: Ok, let's leave it like this for now.

**Review by mhieta — APPROVED** [review]
All good here 👍

**Pull request merged**